clusters/l3-sqnc/monitoring/values.yaml

```diff
@@ -0,0 +1,79 @@
+alertmanager:
+  enabled: true
+  serviceMonitor:
+    selfMonitor: true
+    namespace: monitoring
+  config:
+    receivers:
+      - name: "null"
+      # - 
+      # - name: slack
+      #   webhook_configs:
+      #     - send_resolved: true
+      #       url: <secret>
+prometheus:
+  serviceMonitor:
+    selfMonitor: true
+    namespace: monitoring
+  prometheusSpec:
+    retention: 24h
+    resources:
+      requests:
+        cpu: 200m
+        memory: 200Mi
```

Comment:
We need to set a persistence option here, as well as potentially have these pods autoscaling.  Those default resources look a bit minuscule for a production cluster so we should beef this up somewhat.

Reply:
Running horizontal scaling on prometheus is a mess. It can be configured by using thanos in this chart, but that is a lot of overhead for our needs ATM.

Instead I've set it up as HA, with vertical scaling on the prometheus instances.